File: Mixing/ExtJs/app/model/MxFormularDetail.js
Ext.define('Mixing.model.MxFormularDetail', {
    extend: 'Ext.data.Model',

    fields: [
        { name: 'ElementId', type: 'int' },
        { name: 'Element', type: 'string' },
        { name: 'Target', type: 'float' },
        { name: 'Result', type: 'float' }
    ]
});

File: Mixing/Models/MxFormularDetail.cs
namespace Mixing.Models
{
    public class MxFormularDetail
    {
        public int ID { get; set; }
        public int ElementId { get; set; }
        public string Element { get; set; }
        public double Target { get; set; }
        public double Result { get; set; }
        public double PercentError { get; set; }
    }
}